dxr/hgext/GPL-LICENSE.txt

```diff
@@ -0,0 +1,278 @@
+        GNU GENERAL PUBLIC LICENSE
```

Comment:
We should also note in the top-level license that hgext is an exception and not under the GPL, so there can be no mistaking it.